src/KubernetesClient/Kubernetes.WebSocket.cs

```diff
@@ -226,14 +222,13 @@ public Task<WebSocket> WebSocketNamespacedPodAttachAsync(string name, string @na
 
             uriBuilder.Path += $"api/v1/namespaces/{@namespace}/pods/{name}/attach";
 
-            uriBuilder.Query = QueryHelpers.AddQueryString(string.Empty, new Dictionary<string, string>
-            {
-                { "container", container},
-                { "stderr", stderr ? "1": "0"},
-                { "stdin", stdin ? "1": "0"},
-                { "stdout", stdout ? "1": "0"},
-                { "tty", tty ? "1": "0"}
-            }).TrimStart('?');
+            var query = new StringBuilder();
```

Comment:
I would prefer not to manipulate strings
Maybe copy the code from WebUtilities?

QueryHelpers is more readable

Reply:
How about if I:
1. Call Utilities.AddQueryParameter for each line rather than using Append, or
2. Add Utilities.AddQueryParameters which can taken multiple parameters, and use that?